project/src/test/ui/chat/test-chat-library.gd

```diff
@@ -52,28 +51,28 @@ func before_each() -> void:
 func test_greeting() -> void:
 	PlayerData.chat_history.delete_history_item("chat/gurus750/greeting_001")
 	
-	assert_eq(ChatLibrary.select_from_chat_selectors(_chat_selectors, _state, _filler_ids), "greeting_001")
+	assert_eq(ChatLibrary.select_from_chat_selectors(_chat_selectors, "gurus750", _filler_ids), "greeting_001")
```

Comment:
Extract a utility method in this test to avoid copy/pasting the userid everywhere.

Reply:
Extracted `_select_from_chat_selectors()` utility method.